src/include/daos_prop.h

```diff
@@ -197,6 +199,17 @@ enum {
 	DAOS_RECLAIM_TIME,
 };
 
+enum {
+	DAOS_REINT_MODE_DATA_SYNC = 0,
+	DAOS_REINT_MODE_NO_DATA_SYNC = 1,
+	DAOS_REINT_MODE_INVALID = 2,
```

Comment:
seems "DAOS_REINT_MODE_INVALID" not really needed? as any other value is invalid